<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupervisorMainWindow</class>
 <widget class="QMainWindow" name="SupervisorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SupervisorMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="receivingWrappingLayout">
     <item>
      <widget class="QLabel" name="receivingLabel">
       <property name="text">
        <string>Receiving</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="receivingText">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>581</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sendingWrappingLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sending</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sendingContentLayout">
       <item>
        <widget class="QLineEdit" name="sendingText"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendingButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="connectionWrappingLayout">
     <item>
      <widget class="QLabel" name="connectionWrappingLabel">
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="connoctionContentLayout">
       <item>
        <widget class="QLabel" name="comPortLabel">
         <property name="text">
          <string>ComPort:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comPortComboBox">
         <item>
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyUSB1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyACM0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/dev/ttyACM1</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>BaudRate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudRateComboBox">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectionButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEmbedSupervisor">
    <property name="title">
     <string>EmbedSupervisor</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionSettings"/>
    <addaction name="actionTerminate"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuEmbedSupervisor"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTerminate">
   <property name="text">
    <string>Terminate</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
